src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
add transition properties explicitly (default -all and it works, but it can be hard for old browsers performance)